engine/rpcserver.go

```diff
@@ -16,7 +16,7 @@ import (
 
 	"github.com/gofrs/uuid"
 	grpcauth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
-	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
```

Comment:
After running `buf generate` this change prevent me from building on both windows and wsl. Reverting to v2 fixes this. Is there a reason or something I'm missing with this change?

Reply:
this was my bad, i don't think i correctly updated the `grpc-gateway` dep